storagemarket/types/deal_state.go

```diff
@@ -50,4 +50,7 @@ type ProviderDealState struct {
 	Checkpoint dealcheckpoints.Checkpoint
 	// set if there's an error
 	Err string
+
+	// NBytesReceived is the number of bytes Received for this deal
+	NBytesReceived int64
```

Comment:
We have a ProviderDealInfo object:
```
type ProviderDealInfo struct {
	Deal        *ProviderDealState
	Transferred uint64
}
```

If we want to persist NBytesReceived let's keep it here and get rid of the ProviderDealInfo struct.

If we don't want to persist NBytesReceived let's remove it from here and use ProviderDealInfo instead.